src/mixins/amalgamation-mixin.ts

```diff
@@ -26,91 +26,91 @@ export default class AmalgamationMixin extends Vue {
     this.rule6
   ]
 
-  // *** I'M STILL WONDERING IF I WANT TO USE THESE
-  // readonly isLear = (item: AmalgamatingBusinessIF): boolean => (item?.type === 'lear')
-  // readonly isForeign = (item: AmalgamatingBusinessIF): boolean => (item?.type === 'foreign')
-
   /** True if there a limited company in the table. */
   get isAnyLimited (): boolean {
     return this.getAmalgamatingBusinesses.some(business =>
-      (business.type === 'lear' && business.legalType === CorpTypeCd.BC_COMPANY)
+      (business.type === AmlTypes.LEAR && business.legalType === CorpTypeCd.BC_COMPANY)
     )
   }
 
   /** True if there an unlimited company in the table in Alberta, Nova Scotia or USA. */
   get isAnyUnlimitedAbNsUsa (): boolean {
     return this.getAmalgamatingBusinesses.some(business =>
-      (business.type === 'lear' && business.legalType === CorpTypeCd.BC_COMPANY)
+      (business.type === AmlTypes.LEAR && business.legalType === CorpTypeCd.BC_COMPANY)
     )
   }
 
-  // readonly amalgamationRules = [
-  //   {
-  //     id: 0,
-  //     rule: (v: any) => !!v || 'Required',
-  //     status: AmalgamatingStatuses.ERROR_FOREIGN
-  //   }
-  // ]
```

Comment:
This was an old idea. I prefer the new idea of separate functions for each validation and an array of them (line 20).